autopush/endpoint.py

```diff
@@ -867,3 +868,69 @@ def _load_params(self):
             return json.loads(self.request.body)
         except ValueError:
             return {}
+
+
+class BridgeMessageHandler(RegistrationHandler):
+    cors_methods = "PUT"
+
+    @cyclone.web.asynchronous
+    def put(self, uaid):
+        """HTTP PUT
+
+        Process message states for a UAID
+
+        """
+        self.start_time = time.time()
+        if not self._validate_auth(uaid):
+            return self._write_unauthorized_response(
+                message="Invalid Authentication")
+        params = self._load_params()
+        if 'nak' in params:
+            naks = params['nak']
+            batch = []
+            for nak in naks:
```

Comment:
Yeah, I was debating that a fair bit. Frankly, if I was kicking to a thread, might as well have it just cycle through the naks, decrypt each id and log them. Breaking them into individual thread hits obviously gets through them faster at the cost of potentially chewing up the thread pool if we get a slew of NAKs. 

Meh, I'll stick to one thread to process the stream of naks, and burn the refactor bridge later.
 🐃 🔜 🐩